Gopkg.toml

```diff
@@ -47,6 +47,10 @@
 # because of the lack of semantic versioning in Kubernetes. Therefore we
 # have to pick a certain release, otherwise we (more or less randomly) end
 # up with something older or master.
+[[override]]
+  name = "k8s.io/api"
```

Comment:
i'm surprised client-go didn't need to be updated?

Reply:
Yeah, I found this override block makes it possible. It's more of a temporary fix until all other dependencies update to k8s.io/api 1.15.
https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md#override